Game Dev/Assets/Scripts/UI/CurrencyScript.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
```

Comment:
Pentru ce folosim clasa asta? Ca sa afisam mesajul cu valoarea coin-ului atunci cand este luat de player sau ca sa contorizam banii adunati de player. Daca e a doua varianta nu ar fi mai usor sa avem o variabila in clasa care sa contorizeze banii adunati decat sa dam suma ca parametru?

Reply:
Daca te referi la System.Collections nu facea nimic, nu stiu de ce a fost importata.